src/HttpRequest.h

```diff
@@ -251,6 +264,16 @@ class HttpRequest: public Http::Message
     /// annotations added by the note directive and helpers
     /// and(or) by annotate_transaction/annotate_client ACLs.
     NotePairs::Pointer theNotes;
+
+    Ip::Address client_addr;
```

Comment:
```suggestion
    Ip::Address client_addr; ///< source address of a non-TCP (e.g. ICMP) client
```

And please also document `my_addr`.